Docs 8278 update oas3 docs pa (#265)
mulesoft/docs-api-manager: modules/ROOT/pages/manage-exchange-api-task.adoc

This procedure covers the configuration of the API. After saving the configuration, you can deploy the API.

. In API Manager, select *Manage API* > *Manage API From Exchange*.
+
image::environment.png[]
+

> . In *API Name*, type the first few characters of the API name, and click the search icon.

Reviewer: Second comma not really nec....

. Select the API that you want to manage from the list of available APIs in Exchange. 
+

> The *Asset type*, *API version*, and *Asset version* fields are auto-populated.   

Reviewer: Not hyphenated, per our term preso list:
```suggestion
The *Asset type*, *API version*, and *Asset version* fields are autopopulated.   
```
OR
```suggestion
The *Asset type*, *API version*, and *Asset version* fields are automatically populated.   
```

+

> NOTE: OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager.

Reviewer: Ditto previous edit; GLOBALLY search for and implement name correction.

+

> . In *Managing Type*, choose the type of API endpoint that you want to create:
> * Basic Endpoint: Use this option if you created your API outside API Manager, but want to apply policies or otherwise manage the API using API Manager. For example, if you created the API using Mule Runtime, you do not need a proxy. 
> * Endpoint with Proxy: Use this option if you want to provide additional security for your API. 

Reviewer: GLOBALLY change all bullet-list items to break after the name or first line.
If you have a name, you might consider using a definition list; see style reference.
Note that these are UI labels and so should be in bold; otherwise, I would not capitalize "basic endpoint" or any other endpoint.

```suggestion
. In *Managing Type*, choose the type of API endpoint that you want to create:
* *Basic Endpoint* +
Use this option if you created your API outside of API Manager but want to apply policies or otherwise manage the API using API Manager. For example, if you created the API using Mule runtime engine (Mule), you do not need a proxy. 
* *Endpoint with Proxy* +
Use this option if you want to provide additional security for your API. 
```

+

> . Based the type of endpoint you are creating, specify the application type for a basic endpoint or proxy deployment target for an endpoint with proxy.

Reviewer: ```suggestion
. Based on the type of endpoint you are creating, specify the application type for a basic endpoint or the proxy deployment target for an endpoint with proxy.
```

. If you want to manage the API using Mule 4 or later, select the *Mule version* checkbox.
> In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
+
For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+

> . If you are creating an endpoint with proxy, accept the default Path `/` for the proxy endpoint in the *Path* field, or enter another path. 

Reviewer: ```suggestion
. If you are creating an endpoint with proxy, accept the default path `/` for the proxy endpoint in the *Path* field, or enter another path. 
```

+
For example, changing the path to `/users` makes the REST call to the service at the implementation URI `+http://jsonplaceholder.typicode.com/users+`.
+
. Optionally, expand the *Advanced options* section and specify additional details:
* Specify the validations and an API instance label for the API.

> * If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS), and a port, or accept the defaults.

Reviewer: ```suggestion
* If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
```

* Specify a response timeout, if required. 

> * If you choose Hybrid as the proxy deployment target, select the *Reference user domain* checkbox to use an API gateway domain.

Reviewer: Either lowercase or indicate that this is a UI label by using bold:
```suggestion
* If you choose *Hybrid* as the proxy deployment target, select the *Reference user domain* checkbox to use an API gateway domain.
```
